gen-resourcesdocs/templates/chapter-single-definition.tmpl

```diff
@@ -29,7 +29,7 @@ weight: {{.Metadata.Weight}}
 {{- end}}{{/* range .Fields */}}
 
 {{range .FieldCategories}}
-### {{.Name}} {#{{.Name}}}{{/* explicitly set fragment to keep capitalization */}}
+### {{.Name}} {#{{"-" | regexReplaceAll "[^a-zA-Z0-9]+" .Name }}}{{/* explicitly set fragment to keep capitalization */}}
```

Comment:
@feloy, I am not familiar with this method(from `masterminds.github.io/sprig/strings.html`)?
Are all characters, beside alphanumerical characters, replaced with a hyphen?
In other words, "Not     allowed" --> "Not-----allowed"?

Reply:
More precisely, all series of non alphanumeric characters is replaced by an hyphen. For example
```
Local / Temporary Directory {#Local-Temporary-Directory}
```